src/Jackalope/NodeType/NodeProcessor.php

```diff
@@ -343,19 +344,23 @@ private function validateNamespace($name)
     {
         $alias = null;
         $aliasLength = strpos($name, ':');
-        if (false === $aliasLength) {
+
+        if ($aliasLength === false) {
```

Comment:
its a best practice to have the constant value on the left side and the variable right, to avoid accidentally assigning values. please revert this line